server/src/modes/template/modifierProvider.ts

```diff
@@ -0,0 +1,98 @@
+// tslint:disable: max-line-length
+import { MarkupContent } from 'vscode-languageserver-types';
+
+export interface Modifier {
+  label: string;
+  documentation?: string | MarkupContent;
+}
+
+function genModifier(label: string, documentation?: string | MarkupContent) {
+  return { label, documentation };
+}
+
+const eventModifiers = [
+  genModifier('stop', 'The event propagation will be stopped.'),
+  genModifier('prevent', 'The event will no longer default action.'),
+  genModifier('capture', 'Use capture mode when adding the event listener.'),
+  genModifier('self', 'Only trigger handler if event.target is the element itself.'),
+  genModifier('once', 'The event will be triggered at most once.'),
+  genModifier(
+    'passive',
+    "The scroll event's default behavior (scrolling) will happen immediately, instead of waiting for `onScroll` to complete."
+  ),
+  genModifier(
+    'native',
+    'There may be times when you want to listen directly to a native event on the root element of a component. In these cases, you can use the `.native` modifier for `v-on`'
```

Comment:
```suggestion
    'There may be times when you want to listen directly to a native event on the root element of a component. In these cases, you can use the `.native` modifier.
```

I think that's cleaner.